src/user-event/__tests__/act.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Pressable, Text } from 'react-native';
+import render from '../../render';
+import { userEvent } from '..';
+import { screen } from '../../screen';
+
+test('user event disables act environmennt', async () => {
```

Comment:
test name does not seem to match assertion, could you adjust either of these so they match, alternatively add code comment to explain what is happening

Reply:
I added a comment explaining what happens here